packages/perseus-core/src/widgets/table/index.ts

```diff
@@ -0,0 +1,30 @@
+import type {PerseusTableWidgetOptions} from "../../data-schema";
+import type {WidgetLogic} from "../logic-export.types";
+
+export type TableDefaultWidgetOptions = Pick<
+    PerseusTableWidgetOptions,
+    "headers" | "rows" | "columns" | "answers"
+>;
+
+const defaultRows = 4;
+const defaultColumns = 1;
+
+// initialize a 2D array
+// (defaultRows x defaultColumns) of empty strings
+const answers = new Array(defaultRows)
+    .fill(0)
+    .map(() => new Array(defaultColumns).fill(""));
```

Comment:
I think this is functionally equivalent as:

``` TS
function stringArrayOfSize(size: number): ReadonlyArray<string> {
    return _(size).times(function () {
        return "";
    });
}

const blankAnswers = _(defaultRows).times(function () {
    return Util.stringArrayOfSize(defaultColumns);
});
```

And this way I don't have to use `_` or move a util.

Reply:
Looks synonymous, yeah.  👍